### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by jburel**
Replace travis by GHA

see https://github.com/jburel/conda-zeroc-ice36-python/actions/runs/416036657

### CODE REVIEW

File: .github/workflows/condabuild.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+jobs:
+  build:
+    strategy:
+      matrix:
+        python: [3.6, 3.7, 3.8, 3.9]
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      gradle_version:  # set to empty to build with most recent version of gradle
+      gradle_commands: publishToMavenLocal exec # default is build
```
Comment (manics) [inline .github/workflows/condabuild.yml]:
```suggestion
```

Comment (jburel) [reply .github/workflows/condabuild.yml]: not possible to commit the suggestion via the UI (same text). do you want to push directly to my branch?

Comment (manics) [reply .github/workflows/condabuild.yml]: Oops, I meant to delete the lines not copy them 😄 .

Comment (jburel) [reply .github/workflows/condabuild.yml]: done

**Comment by jburel** [comment]
https://anaconda.org/jburel/zeroc-ice36-python 
Can be used to test the generated output

**Review by manics — APPROVED** [review]
LGTM

**Pull request merged**